routes/index.js

```diff
@@ -7,7 +7,17 @@ const parseRoutes = require('./parseRoutes')
 
 
     , server = (serverType, routesJson, config) => {
-        return serverType.createServer(require('./router.js')(routesJson, config));
+        let serverReturn;
+
+        if (config.key && config.cert) {
+            serverReturn = serverType.createServer({
```

Comment:
Is it needful to document the interface the httpServer should have or is there sufficient convention around Node httpServers (http/https,http2, etc) to have a `createServer` method for, uh, creating the server :)